### PR TITLE
[SQS]  - Added documentation for Queue.set_attribute set_attribute and SQSConnection.set_queue_attribute

### PR DESCRIPTION
**Pull request by PauloMigAlmeida**
As pointed on issue #3128, the documentation regarding to set_attributes wasn't updated. I hope it helps further boto users in the future.

I've also thought about adding a couple more parameters to create_queue method which is currently only taking the visibility_timeout parameter but I guess this is out of the scope of this PR so I'll do it in a different one.

Cheers

### CODE REVIEW

**Comment by signalpillar** [comment]
@pauloubuntu so far generated HTML shows broken formatting for the new docstring. we need to fix this. As an example we can use `get_queue_attributes` method, there is a listing of values but no such cool usage examples as you added. 

Also I didn't find how to build the doc, did you ? 

I added such env to the `tox.ini`
```
[testenv:docs]
changedir = {toxinidir}/docs
commands = make html
setenv =
    PYTHONPATH = {toxinidir}:$PYTHONPATH
```
Now `tox -e docs` results `docs/build/html` folder with HTML files. 

UPD:

Here is a reformatted version for the `set_queue_attributes` of the docstring. Also fixed types for the `attribute` and `queue`.

```
        """
        Set a new value for an attribute of a Queue.

        :type queue: :class:`boto.sqs.queue.Queue`
        :param queue: The SQS queue to get attributes for

        :type attribute: str
        :param attribute: The name of the attribute you want to set.

        :param value: Valid attributes are:

            * `DelaySeconds` - the value must be an integer number of
            seconds from 0 to 900 (15 minutes)::

                queue.set_attribute('DelaySeconds', 900)

            * `MaximumMessageSize` - the value must be an integer number of
            bytes from 1024 (1 KiB) to 262144 (256 KiB)::

                queue.set_attribute('MaximumMessageSize', 262144)

            * `MessageRetentionPeriod` - the value must be an integer number of
            seconds from 60 (1 minute) to 1209600 (14 days)::

                queue.set_attribute('MessageRetentionPeriod', 1209600)

            * `Policy` - the value must be an string that contains JSON formatted
            parameters and values::

                queue.set_attribute('Policy', json.dumps({
                    'Version': '2008-10-17',
                    'Id': '/123456789012/testQueue/SQSDefaultPolicy',
                    'Statement': [
                    {
                        'Sid': 'Queue1ReceiveMessage',
                        'Effect': 'Allow',
                        'Principal': {
                            'AWS': '*'
                        },
                        'Action': 'SQS:ReceiveMessage',
                        'Resource': 'arn:aws:aws:sqs:us-east-1:123456789012:testQueue'
                    }
                ]
                }))

            * `ReceiveMessageWaitTimeSeconds` - the value must be an integer number of
            seconds from 0 to 20::

                queue.set_attribute('ReceiveMessageWaitTimeSeconds', 20)

            * `VisibilityTimeout` - the value must be an integer number of
            seconds from 0 to 43200 (12 hours)::

                queue.set_attribute('VisibilityTimeout', 43200)

            * `RedrivePolicy` - the value must be an string that contains JSON formatted
            parameters and values. You can set maxReceiveCount to a value between 1 and 1000.
            The deadLetterTargetArn value is the Amazon Resource Name (ARN) of the queue that
            will receive the dead letter messages::

                queue.set_attribute('RedrivePolicy', json.dumps({
                'maxReceiveCount': 5,
                'deadLetterTargetArn': "arn:aws:aws:sqs:us-east-1:123456789012:testDeadLetterQueue"
                }))
```
UPD2: could you please also fix doc for the `get_queue_attributes` =) 
There for the `attributes` should be `param` instead of `type` for the long description.

**Comment by pauloubuntu** [comment]
@signalpillar Thanks for reviewing it. 

I've improved formatting as you've pointed out including that last one on get_queue_attributes. 

I think there's always room for doc improvements so feel free to update it whenever you want. :+1:

File: boto/sqs/connection.py [hunk 1]
```diff
@@ -158,6 +158,65 @@ def get_queue_attributes(self, queue, attribute='All'):
...
+        :param queue: The SQS queue to get attributes for
+
+        :type attribute: String
+        :param attribute: The name of the attribute you want to set.
+
+        :param value: The new value for the attribute must be:
+
+            * For `DelaySeconds` the value must be an integer number of
+            seconds from 0 to 900 (15 minutes).
+                >>> queue.set_attribute('DelaySeconds', 900)
```
Comment (signalpillar) [inline boto/sqs/connection.py]: seems like we need to fix these examples to use method `set_queue_attribute` ;)

Comment (pauloubuntu) [inline boto/sqs/connection.py]: Good catch!. I must missed it somehow. 
Done

Comment (signalpillar) [inline boto/sqs/connection.py]: Nice! =)

**Comment by PauloMigAlmeida** [comment]
@danielgtaylor @kyleknap 

Could you guys take a look at this? I see a few guys posting new issues daily because they either didn't find the answer they are looking for on the docs or the docs are outdated/inaccurate. 

That could be a small step but a helpful one for new boto users.

:)

**Comment by kyleknap** [comment]
Thanks! Looks good. Merging.

**Pull request merged**